source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,169 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited : 
+
+
+Avaliable timeouts
+------------------
+
+Captive portal has 3 integrated timeout options :
+
+:**Idle timeout**: Users will automatically disconnected after a defined amount of network inactivity.
```

Comment:
Inside a `:field:` list, you do not need to manually set it bold. So change this to `:Idle Timeout:`, along with the two instances below it.